staging/src/k8s.io/apimachinery/pkg/watch/mux.go

```diff
@@ -220,11 +222,12 @@ func (m *Broadcaster) distribute(event Event) {
 	m.lock.Lock()
 	defer m.lock.Unlock()
 	if m.fullChannelBehavior == DropIfChannelFull {
-		for _, w := range m.watchers {
+		for i, w := range m.watchers {
 			select {
 			case w.result <- event:
 			case <-w.stopped:
 			default: // Don't block if the event can't be queued.
+				glog.Warningf("Dropping event '%#v' (result channel of watcher %d is full)", event, i)
```

Comment:
this might cause warning msg flooding, not only in kubelet. did u try it in your local env?

Reply:
When i tried to figure out the reason why https://github.com/kubernetes/kubernetes/issues/67887 happened, i added some logs here and didn't see it goes here even when kubelet send 2 events per about 12 secs . i will try to look what will happen in proxy/scheduler/node-controller etc.

Reply:
how about changing `glog.Warningf` to `glog.V(4).Infof` ?

Reply:
> how about changing glog.Warningf to glog.V(4).Infof?

i suppose even with a 10 verbosity, it'll still be flooding extremely.  

/cc @liggitt

Reply:
kubernetes by default has 0 verbosity, usually people set higher verbosity for debug purpose.

Reply:
dropping (throttling) events is by design, it is not an error or warning.